instrumentation/rdkafka/example/rdkafka_trace_demonstration.rb

```diff
@@ -0,0 +1,52 @@
+require 'bundler/inline'
+require 'securerandom'
+
+gemfile(true) do
+  source 'https://rubygems.org'
+
+  gem "ffi"
```

Comment:
Was there a reason we had ffi here in the example?

Reply:
I think this was just left over after trying to fix things after my upgrade to macOS Monterey. Will remove.